app/services/feedbacks/contact_to_admin/save.rb

```diff
@@ -0,0 +1,21 @@
+require "dry-monads"
+
+module Feedbacks
+  module ContactToAdmin
+    class Save
+      include Dry::Monads[:result]
+
+      def call(feedback_params)
+        feedback = Feedback.new(feedback_params)
+
+        feedback.save ? Success(feedback) : Failure(feedback)
+      end
+
+      private
+
+      def feedback
+        @feedback ||= Feedback.new(feedback_params)
+      end
```

Comment:
Why you need this?

Reply:
I tested another things in this file and forgot delete redundant